assets/css/styles.css

```diff
@@ -0,0 +1,376 @@
+    letter-spacing: 2px;
+}
+
+.heading h1 {
+    display: flex;
+    justify-content: flex-end;
+    font-size: 30px;
+}
+
+
+/* image fading for project cards */
+
+.img-opacity {
+    opacity: 0.6;
+}
+
+.img-opacity:hover {
+    opacity: 1.0;
+    transition: all 200ms ease-in;
+}
+
+
+/* 
+
+Navigation Styles 
+
+*/
+
+
+/* fixed navigation bar */
+
+.fixed {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1;
+}
+
+.nav-list {
+    padding: 21px;
+    background: var(--clr-nav);
+    box-shadow: 0px 0px 50px var(--clr-nav);
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    align-items: center;
+}
+
+.logo {
+    margin-right: auto;
+}
+
+.logo a {
+    text-decoration: none;
+    align-items: center;
+    font-size: 15px;
+    color: var(--clr-background);
+    letter-spacing: 4px;
+}
+
+
+/* hamburger menu */
+
+#toggle {
+    display: block;
+    cursor: pointer;
+    padding-bottom: 7px;
+}
+
+
+/* navigation button container */
+
+#items {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+/* navgiation buttons/id's for script.js */
+
+#navItem1,
+#navItem2,
+#navItem3 {
+    width: 100%;
+    text-align: center;
+    display: none;
+}
+
+#navItem1 a,
+#navItem2 a,
+#navItem3 a {
+    text-decoration: none;
+    font-size: 15px;
+    color: var(--clr-background);
+    letter-spacing: 3px;
+}
+
+
+/* colour for hovering cursor over each anchor element */
+
+#navItem1 a:hover,
+#navItem2 a:hover,
+#navItem3 a:hover,
+.logo a:hover {
+    color: var(--clr-hover);
+    transition: all 200ms ease-in;
+}
+
+
+/* dropdown list styling for navigation buttons*/
+
+#navItem1.active {
+    border-top: 3px solid var(--clr-hover);
+    border-bottom: 3px solid var(--clr-hover);
+    margin-top: 20px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: flex;
+    justify-content: center;
+}
+
+#navItem2.active,
+#navItem3.active {
+    border-bottom: 3px solid var(--clr-hover);
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: flex;
+    justify-content: center;
+}
+
+
+/* 
+
+Banner Section Styles 
+
+*/
+
+
+/* 
+
+Main Section Styles 
+
+*/
+
+
+/* container styling for about sections */
+
+.container {
+    margin-top: 65px;
+    padding: 0 60px 35px;
+    background-color: var(--clr-blue-yonder);
+}
+
+.about-title {
+    display: flex;
+    width: 100%;
+    padding: 10px 0;
+}
+
+.about-text {
+    display: flex;
+}
+
+.project-container {
+    margin-top: 10px;
+    padding: 0 60px 35px;
+    background-color: var(--clr-blue-yonder);
+}
+
+.project-card-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: var(--clr-blue-yonder);
+    text-align: center;
+}
+
+.project-card {
+    border: 2px solid var(--clr-hover);
+    position: relative;
+    margin: 15px 15px;
+}
+
+.main-project {
+    display: flex;
+    justify-content: center;
+    padding: 25px 0 25px 0;
+}
+
+.flex-row {
+    padding: 0 0 25px 0;
```

Comment:
If you just want to alter padding/margin of just one side, instead of explicitly putting all other sides to 0, just use `padding-bottom` or `padding-left` for example.